core/api/src/services/mongoose/wallets.ts

```diff
@@ -21,6 +21,7 @@ export const WalletsRepository = (): IWalletsRepository => {
     accountId,
     type,
     currency,
+    created_at,
```

Comment:
params shuold be camel case, also, please dont forget to update `resultToWallet` to do the proper "translation"